.github/workflows/ci.yml

```
@@ -0,0 +1,25 @@
+
+# See https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  Package:
+    runs-on: ubuntu-latest
+    container: registry.opensuse.org/yast/head/containers/yast-cpp:latest
+
+    steps:
+
+    - name: Git Checkout
+      uses: actions/checkout@v1
```

We should use the version 2 here, v1 is a bit buggy, see 
- https://github.com/yast/yast-storage-ng/runs/1486235599
- https://github.com/coverallsapp/github-action/issues/55
- https://github.com/ethereumjs/rlp/pull/92


```suggestion
      uses: actions/checkout@v2
```

I'll update the other workflows later...